scripts/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Template build script
+# ---------------------
+
+rm -rf CMakeCache.txt CMakeFiles
+
+export CC=<FIXME: path to C compiler>
+export CXX=<FIXME: path to C++ compiler>
+export MKLROOT=<FIXME: path to the root installation directory of MKL>
+HPX_DIR=<FIXME: path to the root installation directory of HPX>
+BLASPP_DIR=<FIXME: path to the root installation directory of BLASPP>
+LAPACKPP_DIR=<FIXME: path to the root installation directory of LAPACKPP>
```

Comment:
Have you tested it?

Because, I'm not sure that for what concerns `BLASPP` and `LAPACKPP` that uppercase variables work (as I said in my previous comment, I use `blaspp_DIR` and `lapackpp_DIR`)

Reply:
I tested it: works on my laptop.

Reply:
I did a test on my configuration with CMake 3.15.2 on Mac OS.

The variable `_DIR` must be called uppercase or lowercase depending on the name used in `find_package` call, e.g.
- `find_package(BLASPP)` requires `BLASPP_DIR`
- `find_package(blaspp)` requires `blaspp_DIR`

Since the original filename of the BLASPP library CMake config file is `blasppConfig.cmake`, we should keep the lowercase call, which forces us to use the lowercase version of the `blaspp_DIR`. Exactly the same stands for `LAPACKPP`.